app/i18n/es/people.ts

```diff
@@ -227,18 +249,21 @@ const people: People = {
         texts: [
           'Department of Infectious Diseases, St. Marianna University Hospital',
         ],
+        link: 'http://www.k-icnet.org/profile.html',
       },
       {
         img: FumihikoYokota,
         name: 'Fumihiko Yokota',
         role: 'Public Health Advisor',
         texts: ['Associate Professor of Epidemiology, Kyushu University'],
+        link: 'https://q-aos.kyushu-u.ac.jp/en/staff/1770/',
       },
       {
         img: TamerFahmy,
         name: 'Tamer Fahmy',
         role: 'Middle East & Africa Market Advisor',
         texts: ['Former Dräger/Philips MedTech, MBA, SP Jain School of Global Management, BCS Biomedical Engineering, Cairo University'],
```

Comment:
Hm, based on email,

```
Former Dräger/Philips MedTech - Sleep & Respiratory Care
MBA, S P Jain School of Global Management
Bsc Biomedical Engineering, Cairo University
```

it looks like Tamer's description should be 3 separate strings and is missing " - Sleep & Respiratory Care"?
I think "BCS" should be "BSc". Changes made here should also be reflected in the other `people.ts` files in different locales.

<img width="668" alt="Screenshot 2024-12-02 at 8 20 24 PM" src="https://github.com/user-attachments/assets/6142c43f-a340-4c98-91e8-0cbf1e5a713b">

Reply:
I remember amil saying it was "too wordy" so I removed it. BSc makes sense.

Reply:
I add the "Sleep & Respiratory Care" back